Gruntfile.js

```diff
@@ -151,12 +149,21 @@ module.exports = function(grunt) {
       }
     },
 
+    coveralls: {
+      options: {
+        force: true
+      },
+      bwa: {
+        src: 'test/coverage/**/lcov.info'
+      }
+    },
+
     bumpup: ['package.json', 'bower.json']
   });
 
   // Default task.
-  grunt.registerTask('travis', ['default', 'shell:visual']);
+  grunt.registerTask('travis', ['default', 'shell:visual', 'coveralls']);
   grunt.registerTask('visual', ['concat', 'coffeelint', 'coffee', 'uglify', 'shell:visual']);
-  grunt.registerTask('default', ['concat', 'coffeelint', 'coffee', 'uglify', 'karma:minified']);
+  grunt.registerTask('default', ['concat', 'coffeelint', 'coffee', 'karma:unminified', 'uglify', 'karma:minified']);
```

Comment:
Well it will, but I've had some errors on minified code and not on unminified code, and the other way too, so for prod I like to test the code like hell, even if it's a bit redundant...